### ISSUE TITLE
Problems with .malice/config/config.toml

### ISSUE THREAD

**Issue by vmorgo** [maliceio/malice-kibana-plugin#5]
When I run download, install and run the binary version of malice from the command-line, I run into a number of problems as shown in the detailed examples following:

1.  Repeated prompts for newer version of malice config.toml.  Whether I choose yes or no, the existing copy of config.toml is overwritten by one clearly marked as version 0.3.2. 
2.  Prolonged dumpage about the docker container, presumably Elasticsearch is out of date.  5.3, but should be version 5.5.
3.  Attempts edit the file ~/.malice/config/config.toml do not work as any changes I make as suggested previously, e.g., to alter database version from 5.3 to 5.5 and replace "malice" with "5.5" in the UI image name relating to Kibana are immediately overwritten, even if I flag config.toml as read-only for owner, group and world.  The file is reproduced following this list.
4.  Attempts to access the Malice ELK fail completely, producing the errors shown in example 3, below.
5.  Attempts to use the docker version of Malice don't seem to do much better.

The OS is Ubuntu 16.04.03, Mate Edition.  
Malice is running in a KVM virtual machine with 4 cores and 4 GB RAM.

Below is the contents of ~/.malice/config/config.toml.  It is identical to ~/.malice/config/config.toml.backup

```toml
# Copyright (c) 2013 - 2017 blacktop Joshua Maine, All Rights Reserved.
# See LICENSE for license information.

#######################################################################
# MALICE Configuration ################################################
#######################################################################

title = "Malice Runtime Configuration"
version = "v0.3.2"

[author]
  name = "blacktop"
  organization = "MaliceIO"

[web]
  url = "0.0.0.0:80"
  admin_url = "127.0.0.1:3333"

[email]
  host = "smtp.example.com"
  port = 25
  user = "username"
  pass = "password"

[database]
  name = "malice-elastic"
  image = "blacktop/elasticsearch:5.3"
  server = "localhost"
  ports = [9200]
  timeout = 20
  enabled = true

[ui]
  name = "malice-kibana"
  image = "blacktop/kibana:malice"
  server = "localhost"
  ports = [443]
  enabled = true

[environment]
  run = "development"

[docker]
  machine-name = "malice"
  endpoint = "tcp://localhost:2376"
  timeout = 120
  binds = "malice:/malware:ro"
  links = "malice-elastic:elasticsearch"
  cpu = 500000000
  memory = 524288000

[logger]
  filename = "malice.log"
  maxsize = 10
  maxage = 30
  maxbackups = 7
  localtime = false

[proxy]
  enable = false
  http = ""
  https = ""
```

===============================================================
Examples
===============================================================
Example 1:
Attempt to run malice
=================
vmorgo@kvm-malice:~/.malice/config$ malice
INFO[0000] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
no
INFO[0002] Newer version of malice config available: 0.3.2 
INFO[0002] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
no
(Displays malice help screen as expected.)

Example 2:
Attempt to run malice plugin list
=========================
vmorgo@kvm-malice:~/.malice/config$ malice plugin list
INFO[0000] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
yes
INFO[0049] Newer version of malice config available: 0.3.2 
INFO[0049] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
yes
INFO[0051] Newer version of malice config available: 0.3.2 
nsrl
virustotal
shadow-server
fileinfo
yara
avast
avg
bitdefender
clamav
comodo
fprot
fsecure
sophos
floss

Example 3:
Attempt to run Malice Elk.
====================
vmorgo@kvm-malice:~/.malice/config$ malice elk
INFO[0000] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
no
INFO[0004] Newer version of malice config available: 0.3.2 
INFO[0004] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
no
INFO[0009] Newer version of malice config available: 0.3.2 
ERRO[0009] Network malice does not exist, creating now...  env=development exisits=false network=malice
INFO[0010] Created Network: malice                       env=development name=malice
INFO[0010] Created Volume: malice                        env=development
INFO[0011] Elasticsearch Container Started               env=development ip=localhost name="/malice-elastic" port=[9200]
INFO[0011] Waiting for Elasticsearch to come online.     server="http://localhost:9200" timeout=20
INFO[0022] Elasticsearch came online after 11 seconds   
Pulling repository docker.io/blacktop/kibana
ERRO[0023] CreateContainer error = Error: No such image: blacktop/kibana:malice
  env=development
ERRO[0023] StartContainer error = Error response from daemon: page not found
  env=development
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x84b168]

goroutine 1 [running]:
github.com/maliceio/malice/malice/docker/client/container.Start(0xc42039ecc0, 0x0, 0x0, 0x0, 0xc4203355e0, 0xd, 0xc420331d40, 0x16, 0x0, 0x0, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/docker/client/container/start.go:74 +0x468
github.com/maliceio/malice/malice/ui.Start(0xc42039ecc0, 0xaba800, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/ui/ui.go:32 +0x29e
github.com/maliceio/malice/commands.cmdELK(0xc4202eb900, 0xab9120, 0x4)
	/Users/blacktop/go/src/github.com/maliceio/malice/commands/elk.go:19 +0xdb
github.com/maliceio/malice/commands.glob..func4(0xc4202eb900, 0x0, 0xc4202eb900)
	/Users/blacktop/go/src/github.com/maliceio/malice/commands/commands.go:69 +0x4d
github.com/maliceio/malice/vendor/github.com/urfave/cli.HandleAction(0x9c9220, 0xade000, 0xc4202eb900, 0xc420336300, 0x0)
	/Users/blacktop/go/src/github.com/maliceio/malice/vendor/github.com/urfave/cli/app.go:485 +0xd4
github.com/maliceio/malice/vendor/github.com/urfave/cli.Command.Run(0xab899e, 0x3, 0x0, 0x0, 0x0, 0x0, 0x0, 0xac9777, 0x1e, 0x0, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/vendor/github.com/urfave/cli/command.go:207 +0xb72
github.com/maliceio/malice/vendor/github.com/urfave/cli.(*App).Run(0xc4202d4820, 0xc420072080, 0x2, 0x2, 0x0, 0x0)
	/Users/blacktop/go/src/github.com/maliceio/malice/vendor/github.com/urfave/cli/app.go:250 +0x7d0
main.main()
	/Users/blacktop/go/src/github.com/maliceio/malice/main.go:88 +0x584
vmorgo@kvm-malice:~/.malice/config$

Example 4:
Attempt to run Malice scan on a file (eicar.com)
=====================================
vmorgo@kvm-malice:~/.malice/config$ malice scan /home/vmorgo/malicewatch/eicar.com
INFO[0000] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
yes
INFO[0002] Newer version of malice config available: 0.3.2 
INFO[0002] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
yes
INFO[0003] Newer version of malice config available: 0.3.2 
#### File
| Field  | Value                                                            |
| ------ | ---------------------------------------------------------------- |
| Name   | eicar.com                                                        |
| Path   | /home/vmorgo/malicewatch/eicar.com                               |
| Size   | 68B                                                              |
| MD5    | 44d88612fea8a8f36de82e1278abb02f                                 |
| SHA1   | 3395856ce81f2b7382dee72602f798b642f14140                         |
| SHA256 | 275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f |
ERRO[0006] Container is already running...               env=development exisits=true name=shadow-server
ERRO[0006] "container is already running"
goroutine 39 [running]:
github.com/maliceio/malice/malice/errors.CheckErrorWithMessage(0xdf0020, 0xc42036a620, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc4203d8700)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:24 +0x7d
github.com/maliceio/malice/malice/errors.CheckError(0xdf0020, 0xc42036a620, 0xc4203d6cc0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:12 +0x62
github.com/maliceio/malice/plugins.Plugin.StartPlugin(0xc420384550, 0xd, 0x1, 0xc4203846a0, 0x5, 0xc4203807a0, 0x1a, 0xc4203808e0, 0x14, 0xc4203591c0, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:67 +0xa12
created by github.com/maliceio/malice/plugins.RunIntelPlugins
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:109 +0x482

 
ERRO[0006] "Error response from daemon: No container name or ID supplied"
goroutine 39 [running]:
github.com/maliceio/malice/malice/errors.CheckErrorWithMessage(0xdf0020, 0xc4203787a0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42036a6c0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:24 +0x7d
github.com/maliceio/malice/malice/errors.CheckError(0xdf0020, 0xc4203787a0, 0x0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:12 +0x62
github.com/maliceio/malice/plugins.Plugin.StartPlugin.func1(0xc4203d6cc0, 0xc420517d88)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:60 +0x7d
github.com/maliceio/malice/plugins.Plugin.StartPlugin(0xc420384550, 0xd, 0x1, 0xc4203846a0, 0x5, 0xc4203807a0, 0x1a, 0xc4203808e0, 0x14, 0xc4203591c0, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:68 +0xa13
created by github.com/maliceio/malice/plugins.RunIntelPlugins
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:109 +0x482

 
#### VirusTotal
| Ratio      | Link         | API         | Scanned     |
|------------|--------------|-------------|-------------|
| 98% | [link](https://www.virustotal.com/file/275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f/analysis/1502047996/) | Public | 2017-08-06 19:33:16 |

#### NSRL Database
 - Found :white_check_mark:
#### F-PROT
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR_Test_File (exact) | 4.6.5.141 | 20170704 |

2017/08/06 20:01:39 cannot open; magic mime db is already open
time="2017-08-06T20:01:39Z" level=error msg="avast license has expired" 
time="2017-08-06T20:01:39Z" level=error msg="please get a new one here: https://www.avast.com/linux-server-antivirus" 
#### Comodo
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | Malware | 1.1 | 20170704 |

#### Avast
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR Test-NOT virus!!! | 2.2.0 | 20170704 |
#### Bitdefender
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-Test-File (not a virus) | 7.141118 | 20170709 |

#### F-Secure
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-Test-File (not a virus) | 11.10 build 68 | 20170708 |

#### Sophos
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-AV-Test | 5.35.0 | 20170704 |

#### ClamAV
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | Eicar-Test-Signature | 0.99.2 | 20170704 |

time="2017-08-06T20:02:10Z" level=fatal msg="exit status 5" category=av path="/malware/275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f" plugin=avg 
vmorgo@kvm-malice:~/.malice/config$ 

Example 5:
Attempt to scan multiple files in the same directory:
=======================================
(malice appeared to only scan the first file, eicar.com, which is a text file.)
vmorgo@kvm-malice:~/.malice/config$ malice scan /home/vmorgo/malicewatch/*.*
INFO[0000] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
yes
INFO[0003] Newer version of malice config available: 0.3.2 
INFO[0003] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
yes
INFO[0004] Newer version of malice config available: 0.3.2 
#### File
| Field  | Value                                                            |
| ------ | ---------------------------------------------------------------- |
| Name   | eicar.com                                                        |
| Path   | /home/vmorgo/malicewatch/eicar.com                               |
| Size   | 68B                                                              |
| MD5    | 44d88612fea8a8f36de82e1278abb02f                                 |
| SHA1   | 3395856ce81f2b7382dee72602f798b642f14140                         |
| SHA256 | 275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f |
ERRO[0005] Container is already running...               env=development exisits=true name=shadow-server
ERRO[0005] "container is already running"
goroutine 37 [running]:
github.com/maliceio/malice/malice/errors.CheckErrorWithMessage(0xdf0020, 0xc42036efe0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc4204184c0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:24 +0x7d
github.com/maliceio/malice/malice/errors.CheckError(0xdf0020, 0xc42036efe0, 0xc42038fbc0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:12 +0x62
github.com/maliceio/malice/plugins.Plugin.StartPlugin(0xc420386570, 0xd, 0x1, 0xc4203866c0, 0x5, 0xc4203827a0, 0x1a, 0xc4203828e0, 0x14, 0xc420359200, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:67 +0xa12
created by github.com/maliceio/malice/plugins.RunIntelPlugins
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:109 +0x482

 
ERRO[0005] "Error response from daemon: No container name or ID supplied"
goroutine 37 [running]:
github.com/maliceio/malice/malice/errors.CheckErrorWithMessage(0xdf0020, 0xc420386770, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc42036f080)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:24 +0x7d
github.com/maliceio/malice/malice/errors.CheckError(0xdf0020, 0xc420386770, 0x0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:12 +0x62
github.com/maliceio/malice/plugins.Plugin.StartPlugin.func1(0xc42038fbc0, 0xc420201d88)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:60 +0x7d
github.com/maliceio/malice/plugins.Plugin.StartPlugin(0xc420386570, 0xd, 0x1, 0xc4203866c0, 0x5, 0xc4203827a0, 0x1a, 0xc4203828e0, 0x14, 0xc420359200, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:68 +0xa13
created by github.com/maliceio/malice/plugins.RunIntelPlugins
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:109 +0x482

 
#### VirusTotal
| Ratio      | Link         | API         | Scanned     |
|------------|--------------|-------------|-------------|
| 98% | [link](https://www.virustotal.com/file/275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f/analysis/1502047996/) | Public | 2017-08-06 19:33:16 |

#### NSRL Database
 - Found :white_check_mark:
#### Magic
| Field       | Value                  |
|-------------|------------------------|
| Mime        | text/plain        |
| Description | EICAR virus test files |

#### SSDeep
 - `3:a&#43;JraNvsgzsVqSwHq9:tJuOgzsko`

#### TRiD
- Warning: file seems to be plain text/ASCII
- TrID is best suited to analyze binary files!
- 100.0% (.COM) EICAR antivirus test file (7057/5)

#### Exiftool
| Field       | Value                |
|-------------|----------------------|
| error  | exit status 1        |

time="2017-08-06T20:04:25Z" level=error msg="avast license has expired" 
time="2017-08-06T20:04:25Z" level=error msg="please get a new one here: https://www.avast.com/linux-server-antivirus" 
#### F-PROT
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR_Test_File (exact) | 4.6.5.141 | 20170704 |

#### Comodo
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | Malware | 1.1 | 20170704 |

#### Bitdefender
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-Test-File (not a virus) | 7.141118 | 20170709 |

#### Avast
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR Test-NOT virus!!! | 2.2.0 | 20170704 |
#### F-Secure
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-Test-File (not a virus) | 11.10 build 68 | 20170708 |

#### Sophos
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-AV-Test | 5.35.0 | 20170704 |

#### ClamAV
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | Eicar-Test-Signature | 0.99.2 | 20170704 |

time="2017-08-06T20:04:54Z" level=fatal msg="exit status 5" category=av path="/malware/275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f" plugin=avg 
vmorgo@kvm-malice:~/.malice/config$ 

Example 6:  
Attempt to scan two ZIP files.
(Again, it appears only the first file was scanned.)
vmorgo@kvm-malice:~/.malice/config$ malice scan /home/vmorgo/malicewatch/eicar*.zip
INFO[0000] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
no
INFO[0002] Newer version of malice config available: 0.3.2 
INFO[0002] Newer version of malice config.toml available: 0.3.2 
Would you like to update now? (yes/no)
no
INFO[0003] Newer version of malice config available: 0.3.2 
#### File
| Field  | Value                                                            |
| ------ | ---------------------------------------------------------------- |
| Name   | eicarcom2.zip                                                    |
| Path   | /home/vmorgo/malicewatch/eicarcom2.zip                           |
| Size   | 308B                                                             |
| MD5    | e4968ef99266df7c9a1f0637d2389dab                                 |
| SHA1   | bec1b52d350d721c7e22a6d4bb0a92909893a3ae                         |
| SHA256 | e1105070ba828007508566e28a2b8d4c65d192e9eaf3b7868382b7cae747b397 |
ERRO[0005] Container is already running...               env=development exisits=true name=shadow-server
ERRO[0005] "container is already running"
goroutine 24 [running]:
github.com/maliceio/malice/malice/errors.CheckErrorWithMessage(0xdf0020, 0xc42032d660, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420402100)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:24 +0x7d
github.com/maliceio/malice/malice/errors.CheckError(0xdf0020, 0xc42032d660, 0xc420396cc0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:12 +0x62
github.com/maliceio/malice/plugins.Plugin.StartPlugin(0xc420352710, 0xd, 0x1, 0xc420352860, 0x5, 0xc420350b20, 0x1a, 0xc420350c60, 0x14, 0xc420340a40, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:67 +0xa12
created by github.com/maliceio/malice/plugins.RunIntelPlugins
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:109 +0x482

 
ERRO[0005] "Error response from daemon: No container name or ID supplied"
goroutine 24 [running]:
github.com/maliceio/malice/malice/errors.CheckErrorWithMessage(0xdf0020, 0xc420356190, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc420356080)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:24 +0x7d
github.com/maliceio/malice/malice/errors.CheckError(0xdf0020, 0xc420356190, 0x0)
	/Users/blacktop/go/src/github.com/maliceio/malice/malice/errors/errors.go:12 +0x62
github.com/maliceio/malice/plugins.Plugin.StartPlugin.func1(0xc420396cc0, 0xc420477d88)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:60 +0x7d
github.com/maliceio/malice/plugins.Plugin.StartPlugin(0xc420352710, 0xd, 0x1, 0xc420352860, 0x5, 0xc420350b20, 0x1a, 0xc420350c60, 0x14, 0xc420340a40, ...)
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:68 +0xa13
created by github.com/maliceio/malice/plugins.RunIntelPlugins
	/Users/blacktop/go/src/github.com/maliceio/malice/plugins/plugins.go:109 +0x482

 
#### VirusTotal
| Ratio      | Link         | API         | Scanned     |
|------------|--------------|-------------|-------------|
| 87% | [link](https://www.virustotal.com/file/e1105070ba828007508566e28a2b8d4c65d192e9eaf3b7868382b7cae747b397/analysis/1501985116/) | Public | 2017-08-06 02:05:16 |

#### NSRL Database
 - Not Found :grey_question:
time="2017-08-06T20:09:03Z" level=error msg="avast license has expired" 
time="2017-08-06T20:09:03Z" level=error msg="please get a new one here: https://www.avast.com/linux-server-antivirus" 
#### Magic
| Field       | Value                  |
|-------------|------------------------|
| Mime        | application/zip        |
| Description | Zip archive data, at least v1.0 to extract |

#### SSDeep
 - `6:59QykoBveYMTpDAJuOgzskozj9sY//zpW&#43;lIt9xvQ1ioBLl&#43;lE/:5/BGpFAdko9sQcaY9xvwBBRaE/`

#### TRiD
- 100.0% (.ZIP) ZIP compressed archive (4000/1)

#### Exiftool
| Field       | Value                |
|-------------|----------------------|
| ExifToolVersionNumber  | 10.56        |
| FileSize  | 308 bytes        |
| FileType  | ZIP        |
| FileTypeExtension  | zip        |
| MIMEType  | application/zip        |
| ZipBitFlag  | 0        |
| ZipCRC  | 0x31db20d1        |
| ZipCompressedSize  | 184        |
| ZipCompression  | None        |
| ZipFileName  | eicar_com.zip        |
| ZipRequiredVersion  | 10        |
| ZipUncompressedSize  | 184        |

#### F-PROT
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR_Test_File (exact, not disinfectable)&gt; 	/malware/e1105070ba828007508566e28a2b8d4c65d192e9eaf3b7868382b7cae747b397-&gt;eicar_com.zip- | 4.6.5.141 | 20170704 |

#### Comodo
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | Malware | 1.1 | 20170704 |

#### Bitdefender
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-Test-File (not a virus) | 7.141118 | 20170709 |

#### Avast
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR Test-NOT virus!!! | 2.2.0 | 20170704 |
#### F-Secure
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | EICAR-Test-File (not a virus) | 11.10 build 68 | 20170708 |

#### Sophos
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| false |  | 5.35.0 | 20170704 |

#### ClamAV
| Infected      | Result      | Engine      | Updated      |
|:-------------:|:-----------:|:-----------:|:------------:|
| true | Eicar-Test-Signature | 0.99.2 | 20170704 |

#### AVG
| Infected | Result | Engine    | Updated  |
| -------- | ------ | --------- | -------- |
| false    |        | 13.0.3114 | 20170609 |
vmorgo@kvm-malice:~/.malice/config$ 

Example 6:
Attempt to run malice watch.
This works as expected, although, like the previous messages, it throws numerous errors about the docker container (presumably elasticsearch 5.3 is too old, etc., ) as well as what appeared to be numerous expired antivirus definition file licenses.

NOTE:

Previously, you advised editing ~/.malice/config/config.toml, as follows:
"However, just FYI if you go into the malice config you should be able to tell it to use the new versions (just won't have the malice style yet)

~/.malice/config/config.toml

and change it to:

[database]
  name = "malice-elastic"
  image = "blacktop/elasticsearch:5.5"
...

[ui]
  name = "malice-kibana"
  image = "blacktop/kibana:5.5"
..."

I tried this, but every time I run malice, it overwrites the changes, resetting back to "image="blacktop/kibana:malice"" and "image="blacktop/elasticsearch:5.3"".  This happens even if I make both config.toml and config.toml.backup read-only for owner, group and world.  (File is still owned by me, not by "root").

**Comment by blacktop** [comment]
there is most likely a version of malice in your ~/GOPATH/bin folder that has a higher PATH precedence then the newer version of malice you are trying to run

**Comment by blacktop** [comment]
also are you using the latest version here: https://github.com/maliceio/malice/releases/download/v0.3.11/malice_0.3.11_linux_amd64.tar.gz

**Comment by blacktop** [comment]
I am curious if you use the latest version and then use the explicit path to it if it still doesn't work?  I seem to remember having a similar issue as you and it was because I had built it with Go and it was in `~/$GOPATH/bin/malice` which was in a higher PATH than the one I was trying to use.  So when I typed `malice` it was using the old one which would create an old config instead of using the new config.
